src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private StorageDaoImpl storageDao;
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeEach
+    void setUp() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_LoginCorrectLength_Ok() throws RegistrationException {
+        User user = new User("kate43", "tyo7654", 25);
+        User actual = registrationService.register(user);
+        assertEquals(user, actual);
+    }
+
+    @Test
+    void register_LoginShorterThan6_NotOk() {
```

Comment:
```suggestion
    void register_InvalidLoginLength_NotOk() {
```
better avoid using values in the method names